src/components/ui/TimeSlotParticipantsPanel.tsx

```diff
@@ -1,12 +1,14 @@
 import React, { useMemo } from "react";
 import { add, format } from "date-fns";
 import { IoEarthSharp } from "react-icons/io5";
+import { formatTimeZoneTag } from "../../utils/utils";
```

Comment:
just a note but it's interesting how we named these files) idt `/utils/utils` is a good thing to have in code 🤔

Reply:
The first `utils` is the directory "utils" and the second `utils` is the file "utils.ts." I agree that this sounds weird. We can possibly rename the file "utils.ts" to "date-utils.ts" since it currently mainly contains only date-related functions, if you think it is appropriate.

Reply:
Yea I'd say this is better